test/README.md

```diff
@@ -186,6 +177,18 @@ make test
 
 5. Exclude tests:
 
+  * exclude test target in playlist.xml
+
+    Add 
+    ```
+    <disabled>reason for disabled<disabled>
```

Comment:
<disabled>Reason for disabling test, preferably includes issue number<**/**disabled>

May additionally want to include example